File: Lab 3/Apartment.java
public class Apartment extends Accommodation {
    private int floorOfHouse;

    public Apartment(int floorOfHouse, int numberOfRooms, double price, double area, double latitude, double height,
                     double longitude) {
        super(numberOfRooms, price, area);
        this.floorOfHouse = floorOfHouse;
        setLocation(latitude, height, longitude);
    }

    @Override
    public String toString() {
        return "\n\nApartment:" +
                "\nfloorOfHouse = " + floorOfHouse +
                "\nlocation = " + getLocation() +
                "\nnumber of rooms = " + getNumberOfRooms() +
                "\nprice = " + getPrice() +
                "\narea = " + getArea();
    }

    public int getFloorOfHouse() {
        return floorOfHouse;
    }

    public void setFloorOfHouse(int floorOfHouse) {
        this.floorOfHouse = floorOfHouse;
    }
}

File: Lab 13 (lab 4 python)/app/apartment.py
from app.accommodation import Accommodation


class Apartment(Accommodation):

    def __init__(self, number_of_rooms, price, area, latitude, floor_of_house):
        super(Apartment, self).__init__(number_of_rooms, price, area, latitude)
        self.floor_of_house = floor_of_house

    def __str__(self):
        return super(Apartment, self).__str__() + ' floor_of_house = ' \
               + str(self.floor_of_house)